pkgs/applications/misc/ulauncher/default.nix

```
@@ -0,0 +1,70 @@
+{ stdenv, fetchurl, python27Packages,
+  gnome3, gobjectIntrospection, wrapGAppsHook, gtk3, webkitgtk,
+  libnotify, keybinder3, libappindicator, intltool, wmctrl }:
+
+python27Packages.buildPythonApplication rec  {
+  pname = "ulauncher";
+
+  versionNumber = "4.3.2";
+  version = "${versionNumber}.r8";
+
+  src = fetchurl {
+    url = "https://github.com/Ulauncher/Ulauncher/releases/download/${version}/ulauncher_${version}.tar.gz";
+    sha256 = "0494326s84nvyhblcghxzidp2qf0v0l61kyrf8a92qqryqnzmwip";
+  };
+
+  nativeBuildInputs = with python27Packages; [
+    dbus-python
+    distutils_extra
+    gobjectIntrospection
+    gnome3.defaultIconTheme
+    pygtk
+    intltool
+    keybinder3
+    libappindicator
+    mock
+    notify
+    libnotify
+    paramiko
+    pygobject3
+    pyinotify
+    pysqlite
+    python-Levenshtein
+    pyxdg
+    websocket_client
+    webkitgtk
+    wmctrl
+    wrapGAppsHook
+  ];
+
+  pythonPath = nativeBuildInputs;
+
+  propagatedBuildInputs = pythonPath;
+
+  doCheck = false;
+
+  patchPhase = ''
```

```suggestion
  postPatch = ''
```

If you ever add a `patches` they won't be applied because you set `patchPhase`.